optuna/storages/redis.py

```diff
@@ -0,0 +1,562 @@
+import copy
+from datetime import datetime
+import pickle
+
+import redis
+
+from optuna import distributions, exceptions  # NOQA
+from optuna._experimental import experimental
+from optuna.storages import base
+from optuna.storages.base import DEFAULT_STUDY_NAME_PREFIX
+from optuna import structs
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Any  # NOQA
+    from typing import Dict  # NOQA
+    from typing import List  # NOQA
+    from typing import Optional  # NOQA
+
+
+@experimental("1.2.0")
```

Comment:
Please let me suggest the following change because the next release version will be `1.3.0`.
```suggestion
@experimental("1.3.0")
```